docs/proposals/20200506-conditions.md

```diff
@@ -179,11 +179,9 @@ const (
 // Condition defines an extension to status (i.e. an observation) of a Cluster API resource.
 type Condition struct {
    // Type of condition.
-   // +required
```

Comment:
I'm confused. Is this removed because non marker = equired?

Reply:
There are two reasons:
* if a field is not a pointer, doesn't have omitempty and doesn't have +optional it's automatically required
* +required simply doesn't exist (at least not in controller-tools which we use to generate the CRDs)

Reply:
Required gets derived by not having +option, omitempty, or a pointer type.
The +required marker is ignored by controller-gen